met/ancient-near-east-art/harvest_identifiers.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import sys
+import json
+import re
+
+PER_PAGE = 100 # 100 seems to be the max this API supports
+TOTAL_RESULTS = 6175 # found by looking at the totalResults value from the API
+
+# (department=3 corresponds to the Ancient Near Eastern Art department)
+COLLECTION_LISTING_URL = 'https://metmuseum.org/api/collection/collectionlisting?department=3&showOnly=openAccess&perPage=:per_page&offset=:page'
+COOKIES = 'visid_incap_1661922=gIv8fJeISGGcJv0NPFp8CrqodlwAAAAAQkIPAAAAAACA0IKKAUgy2kw+zt//Ldn6njTuEdrNgU5h; incap_ses_116_1661922=bbTncN2klDMCuwKCfzmcAUTCflwAAAAAz69U/RVQ60rLyC5m76xvqQ==; visid_incap_1661977=3hA+zJCRRzuUunC4kljKEWa/dlwAAAAAQUIPAAAAAABop7jZ/Qyw7ASd4P6CBM1D; incap_ses_116_1661977=fBTTDMMiijrHhSh2fzmcAWa/dlwAAAAAwC9Zj2Eg4Bv0E68vqj7+Ow=='
```

Comment:
Sure the cookie is necessary?

Reply:
No, not really.  I'll try again without it.

Reply:
If you don't put in valid cookies here you get the encapsula error.  Same as you would get if you do:

```
curl 'https://metmuseum.org/api/collection/collectionlisting?department=3&showOnly=openAccess&perPage=100&offset=0'
```

If you open this page in a browser you get this interstitial:
<img width="800" alt="screen shot 2019-03-06 at 2 12 09 pm" src="https://user-images.githubusercontent.com/92044/53910625-e3b28b80-4019-11e9-9f15-73114c8d9dc1.png">

Reply:
Weird, I don't get that in the browser but do with curl. So silly cookies it is ...